content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,101 @@
+```
+
+For authors, a challenge was moving metadata around, from storing authors' metadata (online home, Twitter username) inside posts metadata to having authors metadata files under content/authors, but as mentioned earlier, with the `yaml` package you can edit hundreds of Markdown files at once, granted you check the edits carefully.
+
+Read more about [Hugo taxonomies](https://gohugo.io/content-management/taxonomies/), how to write [templates for taxonomy pages](https://gohugo.io/templates/taxonomy-templates/) and [where to save them so Hugo finds them](https://gohugo.io/templates/lookup-order/#examples-layout-lookup-for-taxonomy-list-pages). In our case, the tags and authors pages have different templates: [tags page](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/terms.html), [tag pages](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/taxonomy/tag.html), [authors templates](https://github.com/ropensci/roweb2/tree/master/themes/ropensci/layouts/authors)
+
+## Archive
+
+With precious content scattered over three categories (blog, tech notes, community calls), getting an overview can be tricky hence our adding an [archive page](/archive) featuring all our website content since the beginning of times. Behind the scenes, we [list pages with a date](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html#L29) and [feeds them to an archive partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/archive.html) that we use for the [technotes page](/technotes) and authors pages as well for consistent presentation.
+
+And should you ever get lost, we've now a [cool 404 page](/404), although mentioning it probably lessens its coolness.
+
+# More content
+
+In the previous section we mention the [community calls living in ropensci.org](/commcalls). They used to have their own source and subdomain, but we've transferred them which has several advantages:
+
+* When reading about community calls, you're looking at the same design than when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) list and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each comm call page, the video recording is for instance embedded!
+
+* Community calls now being similar to blog posts and tech notes, they're cross-listed via their tags and authors so are more likely to be found.
+
+# Feed updates
+
+If you want to keep up with our production, you can subscribe to different feeds (or [our newsletter](https://news.ropensci.org/) since it includes mentions of recent posts!).
+
+## All the feeds
+
+Our website has both RSS and JSON feeds. We added a JSON feed because we could and because some people might like that. 
```

Comment:
link to a technology that uses a json feed?

Reply:
Since JSON feeds are so new, I've found nothing, but I've added a link to the official announcement. I've found a few enthusiastic blog posts but they were written right after the announcement.

Reply:
Maybe the format will never get adopted widely.